src/api/public/apidocs-new/paths/build_project_repository_arch_package_jobstatus.yaml

```diff
@@ -0,0 +1,25 @@
+get:
+  summary: Show the build status of a currently running build job.
+  description: >
+    Show the build status of a currently running build job. Shows an empty
+    result if no build job is running.
```

Comment:
A paragraph referring to the schema used for output XML validation (`jobstatus.rng`) should be added. This is already documented in the [apidocs](https://build.opensuse.org/apidocs/index.html#113).

Reply:
Done @eduardoj